.circleci/config.yml

```diff
@@ -5,48 +5,100 @@ orbs:
   azure-cli: circleci/azure-cli@1.1.0
 
 jobs:
+  build:
+    docker: &python3-7
+      - image: circleci/python:3.7.9
+    steps:
+      - checkout
+      - run:
+          name: install pyodbc deps
+          command: |
+            sudo apt-get update && \
+            sudo apt-get install -y curl build-essential g++ apt-transport-https && \
+            sudo apt-get install -y unixodbc-dev && \
+            sudo apt-get install gnupg -y && \
+            gpg --keyserver hkp://keys.gnupg.net --recv-keys 5072E1F5
+      - run:
+          name: install MSFT ODBC Driver
+          command: |
+            curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add - && \
+            curl https://packages.microsoft.com/config/debian/9/prod.list | sudo tee -a /etc/apt/sources.list.d/mssql-release.list && \
+            sudo apt-get update && \
+            sudo ACCEPT_EULA=Y apt-get install -y msodbcsql17 odbc-postgresql mssql-tools
+      - run:
+          name: config for MSFT driver 
+          command: |
+            export PATH="$PATH:/opt/mssql-tools/bin"
+            sed 's/Driver=psql/Driver=\/usr\/lib\/x86_64-linux-gnu\/odbc\/psql/' /etc/odbcinst.ini > /tmp/temp.ini && \
+            sudo mv -f /tmp/temp.ini /etc/odbcinst.ini
+      - run:
+          name: TEMP -- downgrade security
+          command: |
+            sudo chmod +rwx /etc/ssl/openssl.cnf && \
+            # change TLS back to version 1
+            sudo sed -i 's/TLSv1.2/TLSv1/g' /etc/ssl/openssl.cnf && \
+            # allow weak certificates (certificate signed with SHA1)
+            # by downgrading OpenSSL security level from 2 to 1
+            sudo sed -i 's/SECLEVEL=2/SECLEVEL=1/g' /etc/ssl/openssl.cnf 
+      - run:
+          name: install python deps
```

Comment:
We can use `tox` for this, so not needed really. Maybe `pip install dbt` to run the dbt commands